views/forms/ChartShareForm/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { Button, CardActions, Dialog, DialogTitle, DialogContent, DialogActions } from '@mui/material'
+
+import SearchSelect from '../../components/SearchSelect'
+
+const ChartShareForm = ({
+  chartToShare,
+  closeDialog,
+  shareWithUsers,
+  sharedWithOptions,
+  shareFormControl,
+  shareFormValues,
+  clearSelectedUsers,
+  selectAllUsers
+}) => {
+  return (
+    <Dialog
+    open={Boolean(chartToShare._id)}
+    onClose={closeDialog}
+    fullWidth={true}
```

Comment:
do we need this prop..?

Reply:
Yes otherwise it's only as wide as its child elements